sinonjs/sinon
# Use prettier v2

This PR upgrades `prettier` to latest and removes local configuration, adopting `prettier` default options.

#### Solution  - optional

* Upgrade `prettier`
* Remove `.prettierrc`, accepting the default options
* Re-format all JavaScript files using `prettier`
* Configure editors via `.editorconfig`

#### How to verify - mandatory

1. Check out this branch
1. `npm install`
1. `npm run prettier:check`
1. `npm run lint`
1. `npm run test-cloud`
1. Observe that there are no warnings or errors

#### Checklist for author
